spec/omniauth/strategies/redbooth_spec.rb

```diff
@@ -1,102 +1,118 @@
 require 'spec_helper'
 
-describe OmniAuth::Strategies::Redbooth do
+RSpec.describe OmniAuth::Strategies::Redbooth do
```

Comment:
I really prefer to not deal with the global namespace. Latest versions of `rspec` support doing this and in addition they encourage it. So I consider it a good practice.